api/PerformanceNavigationTiming.json

```diff
@@ -38,6 +38,41 @@
           "deprecated": false
         }
       },
+      "activationStart": {
+        "__compat": {
+          "spec_url": "https://wicg.github.io/nav-speculation/prerendering.html#dom-performancenavigationtiming-activationstart",
+          "support": {
+            "chrome": {
+              "version_added": false
```

Comment:
Is this really right? https://mdn-bcd-collector.gooborg.com/tests/api/PerformanceNavigationTiming/activationStart tests true for me in Chrome on macOS.

Reply:
It looks like this feature is only available in macOS, not Windows, so that explains it -- updating the version number!

Reply:
Or actually, hmm...  I can't tell what the story is for this feature, because not a single version of Chrome in BrowserStack seems to support it (Windows or macOS).  It seems that the feature was originally added in [Chrome 93](https://chromiumdash.appspot.com/commit/ae9d670a683ddfec89ea81275de5e326a3013096) and then its exposure conditions changed in [Chrome 97](https://chromiumdash.appspot.com/commit/028804eacce58120729a00c1b1b828a0e3188a30), but Chrome 107 on BrowserStack on macOS Ventura and Windows 10 don't support the feature while Chrome 107 on my local machine with macOS Ventura does.

This will definitely need some more research.

Reply:
This intent to ship was for Android only https://groups.google.com/a/chromium.org/g/blink-dev/c/EdW7O8yG7Jc/m/DpDN4UNZBQAJ

Chromestatus for desktop https://chromestatus.com/feature/5197044678393856